cmd/av/stack.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"emperror.dev/errors"
+	"github.com/spf13/cobra"
+	"strconv"
+)
+
+var stackCmd = &cobra.Command{
+	Use:   "stack",
+	Short: "managed stacked pull requests",
+}
+
+var stackBranchFlags struct {
+	// The parent branch to base the new branch off.
+	// By default, this is the current branch.
+	Parent string
+}
+var stackBranchCmd = &cobra.Command{
+	Use:   "branch <branch name>",
```

Comment:
According to the doc this was supposed to be:
```
av stack create bookstore-backend
```

Not av branch?

Reply:
Aye, I thought `branch` would be more intuitive. `stack create` makes it sound like you need to run the command before starting the stack, whereas `stack branch` is more intuitive I think (you run the command when you want to branch off the current thing you're working on).

Reply:
That's fair. I also like `av branch`, feels similar to `git branch`.